example/speech-demo/run_ami.sh

```diff
@@ -13,7 +13,7 @@ set -u           #Fail on an undefined variable
 . ./cmd.sh
 . ./path.sh
 
-cmd=hostd3.pl
+cmd=run.pl
```

Comment:
Could you add a comment here to say that this is the kaldi wrapper to the distributed job scheduling system. To run locally, use `run.pl`, and refer to kaldi doc for other scheduling systems like sun grid engine and slurm.